src/afl-fuzz-run.c

```diff
@@ -528,11 +535,21 @@ void sync_fuzzers(afl_state_t *afl) {
     closedir(qd);
```

Comment:
`qd` seeems to be no longer used. This breaks travis?

Reply:
right. removed.